data/test-cases/plugins/ms-teams-sync/UI-design-1.0/Disconnecting_user_without_linked_channel.md

```diff
@@ -0,0 +1,48 @@
+---
+# (Required) Ensure all values are filled up
+name: "Updated RHS after disconnecting a user from their account on MM"
+status: Active
+priority: Normal
+folder: UI-design-1.0
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Click on the `disconnect button` in the RHS of MS Teams sync plugin on MM.
+2. Click on `disconnect button` in the `Disconnect Microsoft teams account` modal.
```

Comment:
```suggestion
1. Click on the `disconnect button` in the RHS of MS Teams sync plugin on MM.
2. Click on the `Disconnect button` in the `Disconnect Microsoft Teams Account` modal.
```

Reply:
I think we should try to match the casing as well for keywords mentioned in the markdown